tests/spark/functions/test_clean_names.py

```diff
@@ -0,0 +1,26 @@
+import pytest
+import pandas as pd
+from pyspark.sql import SparkSession
+from sklearn.datasets import load_iris
+
+import janitor.spark
+
+
+@pytest.fixture(scope="module")
+def df():
```

Comment:
Can't figure out how to use hypothesis with spark. Any help would be appreciated.

Reply:
No need to worry about the use of Hypothesis. It's there mainly to generate lots of examples, and help with catching edge-cases that I might not have thought about.